src/main/java/com/mk/cti/model/PhoneResponse.java

```diff
@@ -1,47 +1,102 @@
 package com.mk.cti.model;
 
-import javax.persistence.*;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.GenerationType;
+
+/**
+ * This is a model class for PhoneResponse.
+ */
 
 @Entity
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Class should define a constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.coding.MissingCtorCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Annotation 'Entity' should be on the same line with its target. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.annotation.AnnotationOnSameLineCheck)